src/NUnitConsole/nunit3-console/TestEventHandler.cs

```diff
@@ -133,7 +137,44 @@ private void TestOutput(XmlNode outputNode)
         {
             var testName = outputNode.GetAttribute("testname");
             var stream = outputNode.GetAttribute("stream");
+            if (stream == "Error" && _outWriter != _errWriter)
```

Comment:
If my above comment is correct - then here, could we instead pass a TextWriter into `TestErrorOutput`, and simplify this check?